Documentation/installation/k8s-install-migration.rst

```diff
@@ -194,6 +194,35 @@ Preparation
         $ cat values-initial.yaml
 
 
+For example, Calico has several modes of operation to decide which network interface to use by default on nodes that it manages. This is configured using the Tigera Operator's ``spec.calicoNetwork.nodeAddressAutodetectionV4`` (and respectively ``nodeAddressAutodetectionV6`` for IPv6) parameter.
```

Comment:
Is this supposed to be part of the item (4)? If yes then please indent it to the same level of indentation as the text "Configure any additional ...". Same feedback for the following paragraphs.